xce/lib/XChemPANDDA.py

```diff
@@ -811,6 +859,28 @@ def export_models(self):
             )
         ):
             sample = model[model.rfind("/") + 1 :].replace("-pandda-model.pdb", "")
+
+            # Check if there are non-low confidence models
+            sample_events = inspect_csv[inspect_csv['dtag'] == sample]
+            sample_high_conf_events = sample_events[
+                sample_events['Ligand Confidence'] != "Low"
+            ]
+            if len(sample_events) == 0:
```

Comment:
Why is this done as an export on threshold instead of a filter? I.e. Why do we still export the low confidence events when there are high ones?